dao/src/main/scala/za/co/absa/enceladus/dao/RestDaoFactory.scala

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright 2018-2019 ABSA Group Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package za.co.absa.enceladus.dao
+
+import com.typesafe.config.ConfigFactory
+import za.co.absa.enceladus.dao.menasplugin.MenasCredentials
+
+object RestDaoFactory {
+
+  private val conf = ConfigFactory.load()
+  private val apiBaseUrl = conf.getString("menas.rest.uri")
```

Comment:
Let's get rid of the config dependency and pass this value to the `getInstance` method. This module is used as a library, it shouldn't have internal configurations, especially when it's this easy to avoid it.